src/universal/config/routes.ts

```diff
@@ -105,8 +106,49 @@ export const PrivateRoutes = Object.values(AppRoutes).filter(
   (path) => !PublicRoutes.includes(path)
 );
 
-export const CustomTrackingUrls = {
-  [AppRoutes.ROOT]: 'https://mijn.amsterdam.nl/home',
+type AppRoute = keyof typeof AppRoutes;
+
+export const CustomTrackingUrls: {
+  [key: AppRoute]: (match: Match) => string;
+} = {
+  [AppRoutes['VERGUNNINGEN/DETAIL']]: (match) => {
+    return `/vergunning/${match.params?.title}`;
+  },
+
+  [AppRoutes['INKOMEN/BBZ']]: (match) => {
+    return `/inkomen/bbz/${match.params?.version}`;
```

Comment:
BZZ wordt zonder versie behandeld (ook al zegt code van niet)

```suggestion
    return `/inkomen/bbz`;
```